Jenkinsfile

```diff
@@ -13,11 +13,17 @@ pipeline {
         stage('main') {
             // TODO Windows build in parallel
             steps {
-                sh 'mvn -B clean verify'
+                sh 'mvn -B clean verify -Dset.changelist'
             }
             post {
                 success {
                     junit '**/target/surefire-reports/TEST-*.xml'
+                    dir("$HOME/.m2/repository") {
```

Comment:
`$HOME` is unset, or just `?`. Need to use the `pwd tmp: true` trick. Not sure if that is even possible in Declarative.